alacritty/src/event.rs

```diff
@@ -1546,6 +1505,30 @@ impl Processor {
 
                     info!("Initialisation complete");
                 },
+                // NOTE: don't batch this event to minimize input latency.
```

Comment:
```suggestion
                // NOTE: This event bypasses batching to minimize input latency.
```

Same on the other comment, after that all is good.